quartz-css/buttons/styles/default.scss

```diff
@@ -353,6 +353,19 @@ select {
     8699a6  // loader
   );
 }
+
+.btn-vimeo-black {
+  @extend .btn;
+  @include button-colors(
+    transparentize($astro-granite, .5),    // background
+    transparentize($astro-granite, 0),     // background hover
+    transparentize($white, .2),    // color
+    transparentize($white, 0),     // color hover
+    transparentize($white, .6),    // border color
+    transparentize($white, 0),     // border color hover
+    ffffff      // loader
```

Comment:
Does `button-colors` automatically prefix this last argument with a `#`?

Reply:
Not sure, actually.  I don't think so?  WIll check

Reply:
Checked against crystal, hash free!